ImportError: cannot import name 'logger' from 'evidence'

When running tests:

```
======================================================================================================================== ERRORS ==============================================================================================================================
___________________________________________________________________________________________________ ERROR collecting tests/data_sources/test_cancer_hotspots.py ___________________________________________________________________________________________________
ImportError while importing test module 'evidence-normalization/tests/data_sources/test_cancer_hotspots.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
venv/lib/python3.11/site-packages/_pytest/python.py:493: in importtestmodule
    mod = import_path(
venv/lib/python3.11/site-packages/_pytest/pathlib.py:587: in import_path
    importlib.import_module(module_name)
../../../../.pyenv/versions/3.11.9/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1204: in _gcd_import
    ???
<frozen importlib._bootstrap>:1176: in _find_and_load
    ???
<frozen importlib._bootstrap>:1147: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
venv/lib/python3.11/site-packages/_pytest/assertion/rewrite.py:184: in exec_module
    exec(co, module.__dict__)
tests/data_sources/test_cancer_hotspots.py:4: in <module>
    from evidence.data_sources import CancerHotspots
evidence/data_sources/__init__.py:2: in <module>
    from .cbioportal import CBioPortal
evidence/data_sources/cbioportal.py:10: in <module>
    from evidence import DATA_DIR_PATH, logger
E   ImportError: cannot import name 'logger' from 'evidence' (evidence-normalization/evidence/__init__.py)
_____________________________________________________________________________________________________ ERROR collecting tests/data_sources/test_cbioportal.py ______________________________________________________________________________________________________
ImportError while importing test module 'evidence-normalization/tests/data_sources/test_cbioportal.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
venv/lib/python3.11/site-packages/_pytest/python.py:493: in importtestmodule
    mod = import_path(
venv/lib/python3.11/site-packages/_pytest/pathlib.py:587: in import_path
    importlib.import_module(module_name)
../../../../.pyenv/versions/3.11.9/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1204: in _gcd_import
    ???
<frozen importlib._bootstrap>:1176: in _find_and_load
    ???
<frozen importlib._bootstrap>:1147: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:690: in _load_unlocked
    ???
venv/lib/python3.11/site-packages/_pytest/assertion/rewrite.py:184: in exec_module
    exec(co, module.__dict__)
tests/data_sources/test_cbioportal.py:4: in <module>
    from evidence.data_sources import CBioPortal
evidence/data_sources/__init__.py:2: in <module>
    from .cbioportal import CBioPortal
evidence/data_sources/cbioportal.py:10: in <module>
    from evidence import DATA_DIR_PATH, logger
E   ImportError: cannot import name 'logger' from 'evidence' (evidence-normalization/evidence/__init__.py)
```

I think this forgot to get updated in #49

--- a/evidence/data_sources/base.py
+++ b/evidence/data_sources/base.py
@@ -1,11 +1,14 @@
 """Module for the base data source class"""
 import json
 import hashlib
+import logging
 from pathlib import Path
 from typing import Optional
 
-from evidence import logger
 from evidence.schemas import Response, SourceDataType
+
+
+_logger = logging.getLogger(__name__)
 
 
 class DataSource:
@@ -69,8 +72,7 @@ class DownloadableDataSource(DataSource):
                 if transformed_data_path.exists():
                     data_path = transformed_data_path
                 else:
-                    logger.error(f"The supplied path at {transformed_data_path} "
-                                 f"does not exist.")
+                    _logger.error("The supplied path at %s does not exist.", transformed_data_path)
             else:
                 data_path = self.download_s3_data(src_data_type)
         return data_path

--- a/evidence/data_sources/cancer_hotspots.py
+++ b/evidence/data_sources/cancer_hotspots.py
@@ -3,6 +3,7 @@
 Downloads Site: https://www.cancerhotspots.org/#/download
 Data URL: https://www.cancerhotspots.org/files/hotspots_v2.xls
 """
+import logging
 from os import remove
 import shutil
 from pathlib import Path
@@ -12,9 +13,12 @@ import json
 import boto3
 from botocore.config import Config
 
-from evidence import DATA_DIR_PATH, logger
+from evidence import DATA_DIR_PATH
 from evidence.data_sources.base import DownloadableDataSource
 from evidence.schemas import SourceDataType, SourceMeta, Response, Sources
+
+
+_logger = logging.getLogger(__name__)
 
 
 class CancerHotspots(DownloadableDataSource):
@@ -58,7 +62,7 @@ class CancerHotspots(DownloadableDataSource):
         :return: Path to transformed data file
         """
         data_path = None
-        logger.info("Retrieving transformed cancer hotspots data from s3 bucket...")
+        _logger.info("Retrieving transformed cancer hotspots data from s3 bucket...")
         s3 = boto3.resource("s3", config=Config(region_name="us-east-2"))
         prefix = \
             f"evidence_normalization/cancer_hotspots/{src_data_type.value}_"
@@ -75,13 +79,13 @@ class CancerHotspots(DownloadableDataSource):
                     obj.download_fileobj(f)
                 shutil.unpack_archive(zip_path, self.src_dir_path)
                 remove(zip_path)
-                logger.info("Successfully downloaded transformed Cancer Hotspots data")
+                _logger.info("Successfully downloaded transformed Cancer Hotspots data")
             else:
-                logger.info("Latest transformed Cancer Hotspots data already exists")
+                _logger.info("Latest transformed Cancer Hotspots data already exists")
 
             data_path = transformed_data_path
         else:
-            logger.warning(
+            _logger.warning(
                 "Could not find transformed Cancer Hotspots data in vicc-normalizers "
                 "s3 bucket"
             )

--- a/evidence/data_sources/cbioportal.py
+++ b/evidence/data_sources/cbioportal.py
@@ -1,4 +1,5 @@
 """Module for accessing python client for cBioPortal."""
+import logging
 from typing import Optional
 from pathlib import Path
 import shutil
@@ -7,9 +8,12 @@ import csv
 
 import boto3
 
-from evidence import DATA_DIR_PATH, logger
+from evidence import DATA_DIR_PATH
 from evidence.data_sources.base import DownloadableDataSource
 from evidence.schemas import SourceDataType, SourceMeta, Response, Sources
+
+
+_logger = logging.getLogger(__name__)
 
 
 class CBioPortal(DownloadableDataSource):
@@ -59,7 +63,7 @@ class CBioPortal(DownloadableDataSource):
         """
         is_mutations = src_data_type == SourceDataType.CBIOPORTAL_MUTATIONS
         data_type = "mutations" if is_mutations else "case_lists"
-        logger.info(f"Retrieving transformed {data_type} data from s3 bucket...")
+        _logger.info(f"Retrieving transformed {data_type} data from s3 bucket...")
         s3 = boto3.client("s3")
         zip_fn = "msk_impact_2017_mutations.csv.zip" if is_mutations else "msk_impact_2017_case_lists.csv.zip"  # noqa: E501
         zip_path = self.src_dir_path / zip_fn
@@ -68,7 +72,7 @@ class CBioPortal(DownloadableDataSource):
                                 f"evidence_normalization/cbioportal/{zip_fn}", f)
         shutil.unpack_archive(zip_path, self.src_dir_path)
         remove(zip_path)
-        logger.info(f"Successfully downloaded transformed cBioPortal {data_type} data")
+        _logger.info(f"Successfully downloaded transformed cBioPortal {data_type} data")
         data_path = self.src_dir_path / zip_fn[:-4]
         return data_path
 

